### PR TITLE
Add checks for undefined in controls

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: src/ol/control/rotatecontrol.js [hunk 1]
```diff
@@ -59,9 +59,9 @@ ol.control.Rotate = function(opt_options) {
       ol.css.CLASS_CONTROL;
   var element = goog.dom.createDom('DIV', cssClasses, button);
 
-  var render = options.render ? options.render : ol.control.Rotate.render;
+  var render = options.render !== undefined ? options.render : ol.control.Rotate.render;
```
Comment (fredj) [inline src/ol/control/rotatecontrol.js]: `options.render` is a function, the check for `undefined` is not needed

File: src/ol/control/rotatecontrol.js [hunk 2]
```diff
@@ -59,9 +59,9 @@ ol.control.Rotate = function(opt_options) {
...
 
-  this.callResetNorth_ = options.resetNorth ? options.resetNorth : undefined;
+  this.callResetNorth_ = options.resetNorth !== undefined ? options.resetNorth : undefined;
```
Comment (fredj) [inline src/ol/control/rotatecontrol.js]: `options.resetNorth` is a function, the check for `undefined` is not needed

File: src/ol/control/scalelinecontrol.js [hunk 3]
```diff
@@ -100,7 +100,7 @@ ol.control.ScaleLine = function(opt_options) {
    */
   this.renderedHTML_ = '';
 
-  var render = options.render ? options.render : ol.control.ScaleLine.render;
+  var render = options.render !== undefined ? options.render : ol.control.ScaleLine.render;
```
Comment (fredj) [inline src/ol/control/scalelinecontrol.js]: `options.render` is a function, the check for `undefined` is not needed

**Comment by fredj** [comment]
LGTM

**Comment by gberaudo** [comment]
Thanks @fredj, I removed the 3 unnecessary checks.

**Comment by gberaudo** [comment]
Rebased on master.

**Comment by fredj** [comment]
Thanks

**Comment by gberaudo** [comment]
Thanks for the review @fredj.

**Pull request merged**